src/kOS/Function/Math.cs

```diff
@@ -14,7 +15,7 @@ public override void Execute(SharedObjects shared)
             double argument = GetDouble(PopValueAssert(shared));
             AssertArgBottomAndConsume(shared);
             double result = Math.Abs(argument);
-            ReturnValue = result;
+            ReturnValue = Structure.FromPrimitive(result);
```

Comment:
Chances are that I started doing it this way first before noticing that ReturnValue does it for me.  I don't think it does that much harm, but it can be removed to make it cleaner.

The very first line of FromPrimitive is to abort and return as-is if the argument is already derived from `Structure`, so it's not going to be that expensive.  Still, for the sake of code clarity, go ahead and remove the redundant call.  (Note, *Slap* ReSharper, THIS is what the word "redundant" means - it means doing a thing two times instead of one.  If both the implicit and explicit way of doing something both still result in it happening only exactly once, that's NOT redundant, darn it.)